sheet.go

```diff
@@ -178,36 +178,38 @@ func (f *File) workSheetWriter() {
 // trimRow provides a function to trim empty rows.
 func trimRow(sheetData *xlsxSheetData) []xlsxRow {
 	var (
-		row  xlsxRow
-		rows []xlsxRow
+		row xlsxRow
```

Comment:
This can be simplify to `var row xlsxRow`.

Reply:
Correct. Done.